modules/core/src/main/scala/lucuma/odb/api/schema/ObservationSchema.scala

```diff
@@ -144,18 +146,36 @@ object ObservationSchema {
 
         Field(
           name        = "scienceConfiguration",
-          fieldType   = OptionType(ScienceConfigurationType[F]),
+          fieldType   = OptionType(ScienceConfigurationType),
           description = Some("The science configuration"),
           arguments   = List(ArgumentIncludeDeleted),
           resolve     = c => c.value.scienceConfiguration
         ),
 
+        Field(
+          name        = "itc",
```

Comment:
Are you storing the itc results as part of the observation, Could this be provided by explore?

Reply:
I thought it should be available as part of the API and at any rate, the output is needed in order to generate the sequence so I thought we might as well expose it as an intermediate step.  It seems like it is useful for API users.

I'm not planning on storing it per se, but I think I will stick it in an LRU cache (perhaps expiring after a short while?) or something since it is kind of an expensive thing to compute.  The `ItcSpectroscopyInput` seems like a perfect hash key.